src/app/security-group/shared/validators/cidr.validator.ts

```diff
@@ -0,0 +1,9 @@
+import { AbstractControl, ValidatorFn } from '@angular/forms';
+import { Utils } from '../../../shared/services/utils/utils.service';
+
+export function cidrValidator(): ValidatorFn {
+  return (control: AbstractControl): { [key: string]: any } => {
```

Comment:
I believe that we can use some proper typing { [key: string]: any }

Reply:
Same for other validators

Reply:
changed to ValidationErrors | null